### ISSUE TITLE
Videoner

### ISSUE THREAD

**Issue by ozky** [QupZilla/qupzilla-plugins#62]
Is this plugin support vlc ? i only see in wiki to use it with smplayer.

**Comment by JHooverman** [comment]
This extension was designed to overcome problems with playing videos due to QtWebkit/ flash issues. Videoner might become depracted, because most video sharing platforms support html5 - which is fully supported by QupZilla.
The usage of smplayer is just a recommendation, videoner might also work with vlc.
Please keep in mind, that videoner will only pass video links to an external media player.

**Comment by pejakm** [comment]
Yes, you can. Replace `smplayer` with `vlc`, it should be enough.

**Comment by ozky** [comment]
I get this error when i change smplayer to vlc.
`Your input can't be opened:
VLC is unable to open the MRL 'https://www.youtube.com/watch?v=0hGiW1_aym8'. Check the log for details.`
I wan't to get it work because vlc hits soon to 3.0 and it will add chromecast support,it would be very good to send link to chromecast with vlc.

**Comment by Vistaus** [comment]
@JHooverman Late response, but not everyone like HTML5. Some people prefer external players. I'm not one of them, but I'm just saying that you shouldn't deprecate an extension just like that 'cause there are people who still want to use it.
